### PR TITLE
Fuse Changes to SDK

### PR DESCRIPTION
**Pull request by mul53**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "build": "tsdx build",
     "start": "tsdx watch",
     "test": "tsdx test",
-    "prepublishOnly": "tsdx build"
+    "prepublishOnly": "tsdx build",
+    "install": "tsdx build"
```
Comment (leonprou) [inline package.json]: is  the `intstall` needed?

File: src/constants.ts [hunk 2]
```diff
@@ -8,7 +8,8 @@ export enum ChainId {
   ROPSTEN = 3,
   RINKEBY = 4,
   GÖRLI = 5,
-  KOVAN = 42
+  KOVAN = 42,
```
Comment (leonprou) [inline src/constants.ts]: please add FUSE on top to minimalize changes

File: src/entities/token.ts [hunk 3]
```diff
@@ -79,5 +79,6 @@ export const WETH = {
     'Wrapped Ether'
   ),
   [ChainId.GÖRLI]: new Token(ChainId.GÖRLI, '0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6', 18, 'WETH', 'Wrapped Ether'),
-  [ChainId.KOVAN]: new Token(ChainId.KOVAN, '0xd0A1E359811322d97991E03f863a0C30C2cF029C', 18, 'WETH', 'Wrapped Ether')
+  [ChainId.KOVAN]: new Token(ChainId.KOVAN, '0xd0A1E359811322d97991E03f863a0C30C2cF029C', 18, 'WETH', 'Wrapped Ether'),
+  [ChainId.FUSE]: new Token(ChainId.FUSE, '0x0BE9e53fd7EDaC9F859882AfdDa116645287C629', 18, 'WETH', 'Wrapped Ether')
```
Comment (leonprou) [inline src/entities/token.ts]: please add FUSE on top to minimalize changes

File: package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@uniswap/sdk",
+  "name": "uniswap-fuse-sdk",
```
Comment (leonprou) [inline package.json]: please renamse to @fuseio/fuse-swap-sdk

File: package.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
...
   "license": "MIT",
-  "version": "3.0.3",
+  "version": "3.0.3a4",
```
Comment (leonprou) [inline package.json]: not sure how to handle the versions here..
lets keep the same version as the uniswap team - "3.0.3"

**Review by leonprou — CHANGES_REQUESTED** [review]
Looking good, 
added some comments and q.
Can you also go over the README and change the links and names to the fuse-swap?

File: package.json (hunk 1 above)
Comment (mul53) [reply package.json]: I will remove that, it’s not needed

**Comment by mul53** [comment]
@leonprou made the changes

**Pull request merged**